docs/sources/auth/planning/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Planning IAM integration strategy
+menuTitle: IAM integration
+description: Considerations and .
+aliases:
+  - /docs/mimir/latest/old-architecture/
+  - docs/sources/auth/planning/
+weight: 200
+keywords:
+  - IdP
+  - IAM
+  - Auth
+---
+
+# Planning your IAM integration strategy
+
+The following documentation is meant to shed light on the different authorization and authentications strategies available in Grafana. By doing preparation before implementation, the user will be able to decide which integration strategy suits best for their needs.
+
+Identity and Access Management (IAM) is needed to handle secure access to [Grafana resources]. Integrating Grafana with the proper solution is a key step to allow user access to sensitive data and resources, and to simplify user management and authentication.
+
+## Benefits of having an IAM integration
+
+Integrating with an IAM provider provides the following benefits:
+
+- **User management**: By providing Grafana access to your current user database, you eliminate the overhead of replicating user information and instead have centralized user management for users' roles and permissions to Grafana resources.
+
+- **Security**: IAM solutions provide advanced security features such as MFA, RBCA, and audit trails, which can help to improve the security of your Grafana installation.
+
+- **SSO**: Properly setting up Grafana with your current IAM solution enables users to access Grafana with the same credentials they use for other applications.
+
+- **Scalability**: Adding, updating, or removing users from your user database is immediately reflected in Grafana.
+
+In order to plan an integration with Grafana, assess your organization's current needs, requirements and any existing IAM solutions being used. This should include which set of roles and permissions will be match to each type of user and which groups of users share the same set of permissions and access to shared resources.
+
+After considering needs, chosen IAM solution and taking into consideration the security requirements, the last step is to test it out thoroughly before deploying it to a production environment.
+
+## Where are my users?
+
+The first thing to consider is who are my users? Are the user exclusively within my organization or will they be outside of my organization?
+
+If the users are within my organization, this means that Grafana might be able to integrate with those users by providing connection with to the corresponding Identity Provider.
+
+If the users are outside of my organization, this means that Grafana needs to provide anonymous access, which is not enabled by default.
+
+### Why do I need to organize the users?
```

Comment:
@linoman It seems like this section and the sections that follow are related. Are you providing an overview of the benefits of organizing users here and then providing more detail about each of the options?

Reply:
The planning section will help the user understand that there should be some decision-making aspects to consider before moving on with the installation. This area could be improved by linking this with the references/terminology section. Since that section has not been started, I decided to add unresolved links as placeholders for now.

Do you think of a better way to develop this area for now?